README.md

```diff
@@ -16,7 +16,7 @@ Features include:
 provide the user provider for the core [SecurityBundle](https://symfony.com/doc/current/book/security.html).
 
 **Caution:** This bundle is developed in sync with [symfony's repository](https://github.com/symfony/symfony).
-For Symfony 2.0.x, you need to use the 1.2.0 release of the bundle (or lower)
+For Symfony <2.7, you need to use the 1.3.0 release of the bundle.
```

Comment:
if we were keeping this message, we would have to talk about 1.3.7 rather than 1.3.0 though.

and anyway, leaving the compatibility check to composer is easier to maintain than duplicating this in the readme.